### PR TITLE
Add/update slack actions

### PR DESCRIPTION
**Pull request by warrenvw**
This PR adds 51 missing actions which have been added to Slack API since the last update to this pack.

Additionally, `bin/generate.py` has been created to automate creation of the action files. Currently, the first paragraph in the method documentation is used to populate the description field. This is not always a good idea (e.g., `groups.createChild.yaml`). Future PR's can clean this up.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
 requests>=2.5.0,<2.6
 websocket-client
+beautifulsoup4
+lxml
```
Comment (armab) [inline requirements.txt]: Can you please pin this to some major version, like `requests` so we don't get any sudden surprises when upstream lib indtroduces any breaking changes.

**Comment by armab** [comment]
Please also update [`pack.yaml` version](https://github.com/StackStorm-Exchange/stackstorm-slack/blob/537515437bf5f885f356903b72b31e9dd00e3596/pack.yaml#L10), so we release new version of the pack.

**Comment by warrenvw** [comment]
Thanks for reminding about pack.yaml. Will do.

File: requirements.txt (hunk 1 above)
Comment (warrenvw) [inline requirements.txt]: What about websocket-client?

Comment (armab) [inline requirements.txt]: Yeah, definitely the good practice to do the same for all dependencies.

**Pull request merged**